recipes/isofit/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "isofit" %}
+{% set version = "1.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/adam-erickson/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 06f4ab1319829c61806c228491b8520f61fac52f269d3cbe5543efd9a8d9dd64
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - setuptools
+    - numpy >=1.11
+    - scipy >=1.1.0
+    - numba >=0.38.0
+    - matplotlib >=2.2.2
+    - scikit-learn >=0.19.1
+    - spectral >=0.19
+    - pytest >=3.5.1
+    - pep8 >=1.7.1
+    - python-xxhash >=1.2.0
+
+test:
+  imports:
+    - {{ name }}
+
+about:
+  home: https://github.com/{{ name }}/{{ name }}
+  license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'Imaging Spectrometer Optimal FITting (ISOFIT)'
+  description: |
+    ISOFIT provides routines and utilities for fitting surface, atmosphere
+    and instrument models to imaging spectrometer data
+  dev_url: https://github.com/{{ name }}/{{ name }}
+  doc_url: https://github.com/{{ name }}/{{ name }}/wiki
+  doc_source_url: https://github.com/{{ name }}/{{ name }}/tree/master/docs
+
+extra:
+  recipe-maintainers:
+    - adam-erickson
+    - winstonolson
+    - davidraythompson
```

Comment:
@ davidraythompson Please :+1: that you are ok to be a maintainer

Reply:
I'm okay with this, thanks.